src/Validation/RulesGatherer.php

```diff
@@ -117,6 +124,12 @@ public function extractRulesAndMessages(ProvidesRules $providesRules, array $arg
 
         $messages = $providesRules->messages();
         $this->messages += $this->wrap($messages, $argumentPath);
+
+        $attributes = $providesRules->attributes();
+        $this->attributes = array_merge(
+            $this->attributes,
+            isset($rules[0]) ? $attributes : $this->wrap($attributes, $argumentPath)
```

Comment:
What does this do? Extracting a variable of method with a proper name can make this more cleara.

Also: https://github.com/nuwave/lighthouse/blob/master/CONTRIBUTING.md#multiline-ternary-expressions

Reply:
Changes applied, thank you for your recommendations.